modules/seqexec/server/src/main/scala/seqexec/server/keywords/DhsClientHttp.scala

```diff
@@ -28,6 +29,12 @@ import cats.implicits._
 class DhsClientHttp(val baseURI: String) extends DhsClient {
   import DhsClientHttp._
 
+  private val KeywordsLUT: Map[KeywordName, String] = Map(
```

Comment:
This will adapt a few keyword names that must be sent especially formatted to the DHS

Reply:
An option here would be to add a `dhsName` property to `KeywordName` and push this mapping down into the database. No strong opinion.

Reply:
It could be nullable I suppose so you'd only need to add the exceptions.

Reply:
it should be nullable in that case, only 3 of the nearly 1K keywords need it